docs/_sidebar.md

```diff
@@ -25,21 +25,22 @@
 
   </details>
 
-  <details open>
+  <details>
     <summary><b>Neo 3 Guides</b></summary>
 
   - [Monitoring the Blockchain](neo3_guides/monitoring.md#monitoring-the-blockchain)
   - [Wallets and Accounts](neo3_guides/wallets_and_accounts.md#wallets-and-accounts)
   - [Invoking Smart Contracts](neo3_guides/contract_invocation.md#invoking-smart-contracts)
   - [Transferring Tokens](neo3_guides/token_transfer.md#transferring-tokens)
 
-    <details open>
```

Comment:
same here.

Reply:
just a personal preference. i'll remove the change then.